frame/sudo/src/lib.rs

```diff
@@ -129,7 +134,11 @@ pub mod pallet {
 		/// A sudo-able call.
 		type RuntimeCall: Parameter
 			+ UnfilteredDispatchable<RuntimeOrigin = Self::RuntimeOrigin>
-			+ GetDispatchInfo;
+			+ GetDispatchInfo
+			+ From<frame_system::Call<Self>>;
```

Comment:
I think its not wrong, since we already require the system pallet as config super trait, but also not that nice looking.  
You can put a `where` on the outer benchmark macro.